configs/components/cpp-pcp-client.json

```diff
@@ -1 +1 @@
-{"url": "git://github.com/puppetlabs/cpp-pcp-client.git", "ref": "fbde5d06f846cb649a5f7a296da0e83ef9d86fb4"}
```

Comment:
Do we want the tagged version here?

Reply:
The merge instructions do `for i in {hiera,facter,puppet,marionette-collective,pxp-agent,cpp-pcp-client}; do git checkout master -- configs/components/$i.json;done`, which specifically excludes changes to those json files.

Reply:
Ok, I'll make a note of that for future reference and leave that out of my re-doing of this merge-up PR in a bit.